content/api/commands/mount.md

```diff
@@ -0,0 +1,226 @@
+---
+title: mount
+componentSpecific: true
+---
+
+<Alert type="warning">
+
+Cypress does not have a **cy.mount()** command out-of-the-box. See below for
+info on how to craft your own.
+
+</Alert>
+
+We recommend creating a custom `cy.mount` command which wraps the mount command
+from the framework-specific libraries in your component tests. Doing so offers a
+few advantages:
+
+- You don't need to import the mount command into every test as the `cy.mount`
+  command is available globally
+- You can set up common scenarios that you usually have to do in each test, like
+  wrapping a component in a
+  [React Provider](https://reactjs.org/docs/context.html) or adding
+  [Vue plugins](https://vuejs.org/v2/guide/plugins.html)
+
+If you attempt to use `cy.mount` before creating it, you will get a warning:
+
+<img src="/_nuxt/assets/img/guides/component-testing/cy-mount-must-be-implemented.png" alt="cy.mount must be implemented by the user." />
+
+Let's take a look at how to implement the command.
+
+## Creating a New `cy.mount` Command
+
+To use `cy.mount` you will need to add a
+[custom command](/api/cypress-api/custom-commands) to the commands file. Below
+are examples that you can start with for your commands:
+
+:::react-vue-example
+
+```js
+import { mount } from '@cypress/react'
```

Comment:
Can you add a line break between any `import` statements and the remaining code in these examples? I think it might make them a little easier to read.